js/user_form.js

```diff
@@ -0,0 +1,41 @@
+import {sendData} from './api.js';
+import {adForm} from './data.js';
+
+const ERROR_MESSAGE_TEMPLATE = document.querySelector('#error').content.querySelector('.error');
+const SUCCESS_MESSAGE_TEMPLATE = document.querySelector('#success').content.querySelector('.success');
+
+const close = () => {
+  document.querySelector('.user-message').remove();
+  document.removeEventListener('keydown', add);
+};
```

Comment:
Чтобы избавиться от ошибки линтера можешь поместить **close** внутрь **add**. Опять таки **add** нужно переименовать, иначе не ясно что добавляется

```suggestion
const onUserMessageKeydown = (e) => {
  e.preventDefault();
  if ((e.key === 'Escape' || e.key === 'Esc') || e.type === 'click') {
    document.querySelector('.user-message').remove();
    document.removeEventListener('keydown', add);
  }
};
```

Также смотри [критерий Д24](https://up.htmlacademy.ru/javascript/23/criteries#d24)

Reply:
@PrijGalina 

**UPD**
Чтобы таких ошибок не возникало нужно обновить линтер до последней версии (4.0.2) и использовать функциональное объявление для функций, которые используются до объявления, чтобы функция всплывала, **function close(){}** 

На лекции должны были об этом говорить. 

`npm install eslint-config-htmlacademy@latest -DE`